### ISSUE TITLE
"Non-reversible reg-exp portion: '(?i'" exception

### ISSUE THREAD

**Issue by acerix** [linuxsoftware/ls.joyous#3]
Getting this on default install:

```
Exception Value: | Non-reversible reg-exp portion: '(?i'
/django/utils/regex_helper.py in normalize, line 126
```

Which is referring to this regex:

https://github.com/linuxsoftware/ls.joyous/blob/master/ls/joyous/models/calendar.py#L137

Error goes away if I remove `(?i)` from the regex.

**Comment by acerix** [comment]
In Django 2.1, `(?i)` (IGNORECASE) needs to be at the end of the regex:

https://stackoverflow.com/a/51773173

**Comment by linuxsoftware** [comment]
Fixed in 27c2fc65ff64a62d50facc6bf2894acf3432210e
